src/Shared/LayoutRenderers/AspNetRequestContentLengthLayoutRenderer.cs

```diff
@@ -12,7 +12,7 @@ namespace NLog.Web.LayoutRenderers
     /// ${aspnet-request-contentlength}
     /// </remarks>
     [LayoutRenderer("aspnet-request-contentlength")]
-    public class AspNetRequestContentLength : AspNetLayoutRendererBase
+    public class AspNetRequestContentLengthLayoutRenderer : AspNetLayoutRendererBase
```

Comment:
Though it seems tempting, then you can repair past mistakes with major-version-bump (Ex. NLog.Web v6)

Please revert, then we can make an issue for v6 about fixing this.